UI/aoc-app/src/Components/CreateProfile.jsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import { Button, Col, Form, FormGroup, Label, Input, FormText } from 'reactstrap';
+
+const CreateProfile = () => {
+    return (
+        <Form>
+            <FormGroup row>
+            <Label for="name" sm={2}>Complete Profile</Label>
```

Comment:
I see label for = "name" twice -- is label supposed to be unique or is this intentional?

Reply:
Good question! I was wondering the same. They're part of a different `FormGroup` and thus have a different parent, so they should still be unique hierarchically. I left the names to be the same since they're being used for each reference. We can revisit later if we need these to be unique for any reason, but I don't see that use case currently.